modules/core/src/components/declarative-ui/xviz-panel.js

```diff
@@ -36,7 +36,15 @@ const DEFAULT_COMPONENTS = {
   plot: XVIZPlot,
   video: XVIZVideo,
   table: XVIZTable,
-  treetable: XVIZTable
+  treetable: XVIZTable,
+
+  // Normalize enums to uppercase
+  CONTAINER: XVIZContainer,
```

Comment:
This is technically public API (users can override default components with their own). Can we cast component type to lowercase in `_renderItem` instead?

Reply:
I am doing this just for the default components then OK.